src/modules/featurepopup/featurepopup-directive.js

```diff
@@ -1,6 +1,7 @@
 import './module.js';
```

Comment:
is there no need to import `util` for using `$olOn` here?

Reply:
good catch